src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -119,16 +128,16 @@ private String getUsername(String userId) {
     return (String) currentUser.getProperty("username");
   }
 
-  // TODO: @salilnadkarni update checkStatus / updateStatus to use Datastore
-  private boolean checkStatus(String userId) {
-    if (!status.containsKey(userId)) {
-      status.put(userId, false);
-    }
-    return status.get(userId) == true;
+  private boolean checkGuessStatus(String userId, Entity currentRound) {
```

Comment:
I agree with @guavafanclub  comment that this can still be more clear. checkCorrectGuess or something

Status is too vaguE IMO